ODF-community.french.md

```diff
@@ -0,0 +1,526 @@
+ouvert<br/><br/>
+ • Publier votre processus de conception<br/><br/>
+
+<strong>Batir une diversité de points de vue + un environement inclusif<strong><br/>
+ • Solliciter les usagers internes et les parties prenantes 
+très tôt, en particulier ceux qui pourraient ne pas être d'accord.<br/><br/>
+ • Chercher des perspectives variées et sous-représentées (emplacement
+géographique, ethnie, départements, niveau hiérarchique, genre, age, etc. )<br/><br/>
+ • Promouvoir la collaboration et fournir des canaux de feedback<br/><br/>
+ • Prendre en compte les risques, les limites et les impacts culturels
+potentiels, en particulier pour les problèmes qui sont des controverses
+historiques<br/><br/>
+</td>
+<td>
+ • Quel est l'impact potentiel sur l'organisation ? Sur la culture ?<br/><br/>
+ • Que faut-il inclure dans le planning ?<br/><br/>
+ • Qui est concerné par le problème que nous essayons de régler?<br/><Br/>
+ • A qui devons ou voulons demander de l'aide ?<br/><br/>
+ • Qui d'autres pourrait être impacté ?<br/><br/>
+ • Qui a résolu un problème similaire ?<br/><br/>
+ • Qui va probablement être en désaccord, s'opposer, rejeter ou de se
+désengager ?  Qui d'autre devons-nous prendre en compte?
+</td>
+<td rowspan=3>
+Il y a un poignée de problèmes qui provoquent en général de la controverse ou de
+l'énervement chez Red Hat, notamment :<br/><br/>
+ • Les décisions, les politiques ou changement qui impactent les collaborateurs
+, par exemple les rémunérations et le programme de bien-être.<br/><br/>
+ • Les changements dans l'environement de travail des collaborateurs<br/><br/>
+ • L'implementation d'une technologie propriétaire<br/><br/>
+ • L'utilisation de formats proprietaires<br/><br/>
+ • Le partage de données et la vie privée<br/><br/>
+
+Si votre projet touche à l'un de ces thèmes, ajoutez des étapes supplémentaires
+pour garantir que votre processus est ouvert, inclusif et transparent.
+</td>
+</tr>
+<tr>
+<td><strong>Points clés</strong></td>
+</tr>
+<tr>
+<td>
+ • Confidentialité, vie privée, et contraintes légales<br/><br/>
+ • Controverses potentielles<br/><br/>
+ • Impact sur la culture Red Hat et les futures décisions<br/><br/>
+ • Roles + responsibilités (modèle OPT model: https://github.com/red-hat-people-team/opt-model/) <br/><br/>
+ • Où publier ?
+</td>
+</tr>
+</table>
+
+<br/>
+
+Phase or activity: Planifier, Rechercher <a name="plan-research"></a>
+----------------------
+
+_page 8 sur 15_
+
+<table>
+<tr>
+<td colspan=2 width="66%"><strong>Étapes pour devenir ouverts</strong></td>
+<td><strong>Questions à poser</strong></td>
+</tr>
+<tr valign="top">
+<td rowspan=3 width=33%><strong>Solliciter usagers + collaborateurs</strong><br/>
+ • Rassembler des retours auprès d'usagers internes et de personnes qui vont
+vous aider (questionnaires, entretiens, groupes témoins, etc.)<br/><br/>
+ • Faciliter la participation + les gestion de projet. Demander aux usagers
```

Comment:
Either 'la gestion', or we need to use the plural form for "gestions". I would favor using "la gestion" since we say "la partitipation".